docs/release-notes.md

```diff
@@ -33,15 +33,26 @@
 
 # Release notes
 
+## Trino Gateway 7 (19 Mar 2024)
```

Comment:
I would recommend mentioning logger change because it affects config files.

Reply:
Good point .. let me add that.